qa/coccinelle/banned-functions.cocci

```diff
@@ -3,7 +3,7 @@ identifier i;
 position p1;
 @@
 
-\(strtok@i\|sprintf@i\|strcat@i\|strcpy@i\|strncpy@i\|strncat@i\|strndup@i\|strchrnul@i\|rand@i\|rand_r@i\|memmem@i\|index@i\|rindex@i\|bzero@i\)(...)@p1
```

Comment:
memmem was banned as mingw doesn't have it

Reply:
I suppose we can have a ```memmem``` implementation for MinGW that just uses our BasicSearch.

Have to then: check for memmem in configure. If it exists: ok. If it doesn't: use wrapper.